public/index.html

```diff
@@ -0,0 +1,373 @@
+                    <li><a href="" class="socicon linkedin"></a></li>
+                    <li><a href="" class="socicon gplus"></a></li>
+                </ul>
+                <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard.</p>
+            </li>
+        </ul>
+        <h2 class="subheader">CLIENTS</h2>
+        <div class="client-wrapper">
+            <a href="" class="socicon arrowLeft"></a>
+            <div class="client-section">
+                <div class="client-pic-wrapper">
+                    <img class="client-pic" src="./imgs/mattbroderic.png" alt="Picture of Matthew Broderic">
+                </div>
+                <div class="client-text-wrapper">
+                    <div class="client-name-title">
+                        <h3 class="client-name">Matthew Broderic</h3><h4 class="client-title">(CEO Hopicola Inc.)</h4>
+                    </div>
+                    <p class="client-text">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when lorem ipsum is best for you.</p>
+                </div>
+            </div>
+            <a href="" class="socicon arrowRight"></a>
+        </div>
+    </section>
+    <section class="contact-section contact-us">
+        <h1 id="contact">CONTACT</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <div class="map-form-container">
+            <form class="map-form">
+                <input type="text" placeholder="Company name" id="form_companyname">
+                <input type="email" placeholder="Your@mail.com" id="form_email">
+                <textarea name="message" id="contact-form" placeholder="Message..."></textarea>
+                <button id="contact-submit" class="message-button" type="submit">SEND MESSAGE</button>
+            </form>
+            <div class="map"></div>
+        </div>
+    </section>
+    <footer>
+        <p>Theme by <a href="">Pierre Borodin</a> &#169; copyright 2012</p>
+        <ul class="social-media">
+            <li><a href="" class="socicon facebook"></a></li>
+            <li><a href="" class="socicon twitter"></a></li>
+            <li><a href="" class="socicon dribble"></a></li>
+            <li><a href="" class="socicon linkedin"></a></li>
+            <li><a href="" class="socicon gplus"></a></li>
+        </ul>
+    </footer>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script type="text/javascript" src="main.js"></script>
+    <script type="text/javascript" src="slick/slick.min.js"></script>
```

Comment:
The `type` attribute isn't necessary for `<script>` tags, and in fact you omitted it on 368. Be consistent.